test/components/generators/chat/test_openai.py

```diff
@@ -872,7 +872,7 @@ def test_live_run_with_tools_streaming(self, tools):
         assert len(results["replies"]) == 1
         message = results["replies"][0]
 
-        assert not message.texts
+        assert message.texts == [""]
```

Comment:
I'm quite confident that this should be `None` for consistency with other test cases and Chat Generators.

Maybe you can do something like
`return ChatMessage.from_assistant(text=text or None, tool_calls=tool_calls, meta=meta)`?

Reply:
Yeah for sure I can update that.